docs/website/docs/general-usage/schema-evolution.md

```diff
@@ -0,0 +1,205 @@
+
+We’ll review some examples here and figure out how `dlt` creates initial schema and how normalisation works. Let's begin by creating a pipeline that loads the following data:
+
+```python
+{
+    "organization": "Tech Innovations Inc.",
+    "address": {
+        'building': 'r&d', 
+        "room": 7890,
+    },
+    "Inventory": [
+        {"name": "Plasma ray", "inventory nr": 2411},
+        {"name": "Self-aware Roomba", "inventory nr": 268},
+        {"name": "Type-inferrer", "inventory nr": 3621}
+    ]
+}
+```
+
+The schema of data above is loaded to the destination as follows:
+<iframe width="560" height="315" src='https://dbdiagram.io/e/65e5c68bcd45b569fb7805e8/65e7ff92cd45b569fba253d9'> </iframe>
+
+### What did the schema inference engine do?
+
+As you can see above the `dlt's` inference engine generates the structure of the data based on the source and provided hints. It normalizes the data, creates tables and columns, and infers data types.
+
+For more information, you can refer to the **[Schema](https://dlthub.com/docs/general-usage/schema)** and **[Adjust a Schema](https://dlthub.com/docs/walkthroughs/adjust-a-schema)** sections in the documentation.
+
+## Evolving the schema
+
+For a typical data source schema tends to change with time, and `dlt` handles this changing schema seamlessly.
+
+Let’s add the following 4 cases:
+
+- A column is added : a field named “CEO” was added.
+- A column type is changed: Datatype of column named “inventory_nr” was changed from integer to string.
+- A column is removed: a field named “room” was commented out/removed.
+- A column is renamed: a field “building” was renamed to “main_block”.
+
+Please update the pipeline for the cases discussed above.
+```python
+{
+    "organization": "Tech Innovations Inc.",
+    # Column added:
+    "CEO": "Alice Smith", 
+    "address": {
+        # 'building' renamed to 'main_block'
+        'main_block': 'r&d', 
+	      # Removed room column
+        # "room": 7890,  
+    },
+    "Inventory": [
+        # Type change: 'inventory_nr' changed to string from int
+        {"name": "Plasma ray", "inventory nr": "AR2411"}, 
+        {"name": "Self-aware Roomba", "inventory nr": "AR268"},  
+        {"name": "Type-inferrer", "inventory nr": "AR3621"}  
+    ]
+}
+```
+
+Let’s load the data and look at the tables:
+<iframe width="560" height="315" src='https://dbdiagram.io/e/65e80303cd45b569fba28e9d/65e80556cd45b569fba2b8ab'> </iframe>
+
+What happened?
+
+- Added column:
+    - A new column named “ceo” is added  to the “org” table.
+- Variant column:
+    - A new column named `inventory_nr__v_text` is added as the datatype of the column was changed from “integer” to “string”.
+- Removed column stopped loading:
+    - New data to column “room” is not loaded.
+- Column stopped loading and new one was added:
+    - A new column “address__building” was added and now data will be loaded to that and stop loading in the column “address__main_block”.
```

Comment:
The column `inventory_nr__v_text` is formatted as code while the rest are in "". I would format the columns as code or bold in this block as it makes for an easier reading experience.

Reply:
also always look in deploy preview
![image](https://github.com/dlt-hub/dlt/assets/20367975/f8b638f2-1407-41c9-b583-186a7832ff23)

Reply:
Yes changed all column names as code.